src/runners/run-local-docker.ts

```diff
@@ -98,6 +98,13 @@ export default class LocalDockerRunner implements UIXRunner {
 		if (verboseArg) args.push("-v")
 		if (clear) args.push("--clear");
 
+		const ports = []
+
+		if (params.inspect!=undefined) {
+			args.push("--inspect=0.0.0.0:9229");
+			ports.push(`${params.inspect||'9229'}:${params.inspect||'9229'}`)
+		}
```

Comment:
We run the inspector on 9229 if we set it explicitly but expose the port for example on 1234 - so `1234:1234`?
Shouldn't be the latter (after `:`) be strictly set to 9229 in any case?

```ts
ports.push(`${params.inspect||'9229'}:9229`)
```

Reply:
yes, but probably it's better if the --inspect parameter also uses the dynamic port